### PR TITLE
Sim footsensor phase reset and PIDs

### PR DESCRIPTION
**Pull request by Flova**
# Summary
- Use foot pressure sensors in sim
- Use PID controllers in sim

### CODE REVIEW

File: bitbots_motion/bitbots_quintic_walk/config/walking_wolfgang_simulator.yaml [hunk 1]
```diff
@@ -75,23 +75,23 @@ walking:
...
         pitch:
-          p: 0.0
+          p: 0.0035
           i: 0.0
-          d: 0.0
+          d: 0.004
           i_clamp_min: 0.0
           i_clamp_max: 0.0
-          antiwindup: False
+          antiwindup: false
```
Comment (jaagut) [inline bitbots_motion/bitbots_quintic_walk/config/walking_wolfgang_simulator.yaml:94]: Why did you change this? Seems like other cases are written as `False`.

Comment (Flova) [reply bitbots_motion/bitbots_quintic_walk/config/walking_wolfgang_simulator.yaml:94]: I think I just copied it from the real robots config file.
